experimental/class-static-block.md

```diff
@@ -0,0 +1,22 @@
+# [class static initialization blocks][proposal-class-static-block]
+
+## ClassBody
+
+```js
+extend interface ClassBody {
+    body: [ MethodDefinition | PropertyDefinition | StaticBlock ];
+}
+```
+- `body` has at most one `StaticBlock`.
```

Comment:
I'm somewhat tempted to suggest moving it out in a separate property in that case... but I can see the argument for preserved ordering being useful so idk. I guess we can leave it for later.

Reply:
Yeah I agree. The static block is sort-of like the `constructor`, in a way that it has special semantics but can mix with other class elements in any order. To align to current situation that `constructor` is a `MethodDefinition` in `body`, I go with extending current body.